content/living_abroad.md

```diff
@@ -1,66 +1,49 @@
-
-
-##Taking your pension money when you live abroad
-
-You can have your pension money paid into a UK bank account. Check with your bank about what charges may apply, eg:
+##Tax when taking your pension abroad
 
+You might have to pay [UK tax on your pension](https://www.gov.uk/tax-uk-income-live-abroad/overview). 
 
-* if you take out your money abroad using a debit card
-* if you transfer your pension money to another account
+You might also have to pay tax in the country where you're resident. If that country has a double taxation agreement with the UK, you might not have to be [taxed twice - in the UK and abroad](https://www.gov.uk/tax-uk-income-live-abroad/taxed-twice). 
 
+If you [plan to retire abroad](https://www.gov.uk/moving-or-retiring-abroad) you must tell HM Revenue and Customs so that you pay the right tax on your pension.
 
-UK pension providers don’t usually pay your pension straight into an overseas bank account. If they do, they may charge you. The exchange rate between the UK and the country you move to can also affect how much money you get.
-
-
-##Don’t forget about tax
-
-
-You must tell HM Revenue and Customs if you [plan to retire abroad](https://www.gov.uk/moving-or-retiring-abroad) so that you pay the right tax on your pension. The amount of tax you pay and which country you pay tax to depend on many things.
-
-If you move abroad before you start taking your pension, [overseas tax laws](https://www.gov.uk/tax-right-retire-abroad-return-to-uk) may prevent you from taking anything tax free.
+If you move abroad before you start taking your pension, [overseas tax laws](https://www.gov.uk/tax-right-retire-abroad-return-to-uk) may prevent you from taking anything tax free. 
 
 
 ##Your State Pension
 
-Find out what happens to your [State Pension if you retire abroad.](https://www.gov.uk/state-pension-if-you-retire-abroad)
-
+You can get your [State Pension abroad](https://www.gov.uk/state-pension-if-you-retire-abroad) if you qualify. 
 
 ##Getting more help
 
-Contact [The Pensions Advisory Service](http://www.pensionsadvisoryservice.org.uk) for free and impartial advice.
+Contact [The Pensions Advisory Service](http://www.pensionsadvisoryservice.org.uk) for free and impartial information on how to take your pension abroad.
+
+The [International Pension Centre](https://www.gov.uk/international-pension-centre) can help with your State Pension abroad.
 
-You can find a [financial adviser](/shop-around#getting-financial-advice) to help you choose the right pension option when moving abroad. You may be charged a fee for this advice.
+You can find a [financial adviser](https://www.pensionwise.gov.uk/shop-around#getting-financial-advice) to help you choose the right pension option when moving abroad. You may be charged a fee for this advice.
```

Comment:
This should be a relative link so that it works on development and staging environments ... `[financial adviser](/shop-around#getting-financial-advice)`